server/methods/core/cart.js

```diff
@@ -101,6 +101,12 @@ function removeShippingAddresses(cart) {
   }, {
     $set: { shipping: cartShipping }
   });
+
+  // Calculate discounts
+  Hooks.Events.run("afterCartUpdateCalculateDiscount", cart._id);
```

Comment:
Also here, why do we tie this to specific events, what if I want to do something else besides calculate discounts and taxes?